piksi_multi_cpp/src/device/device_usb.cc

```diff
@@ -104,15 +104,26 @@ Identifier DeviceUSB::identifyPiksiAndGetSerialNumber(struct sp_port* port) {
     ROS_ERROR_STREAM("Cannot get vendor ID and product ID." << result);
     return nullptr;
   }
+  std::tuple<uint16_t, uint16_t> usb_vidpid = {usb_vid, usb_pid};
+  ;
```

Comment:
This is to tick the processor once isn't it? Maybe comment.

Reply:
uups nope its a type :-)